src/components/PlayerInfo.vue

```diff
@@ -3,7 +3,7 @@
     <div class="text">
       <b>Player {{player.player + 1}}</b> - {{faction}} - {{data.victoryPoints}}vp <span v-if="passed">(passed)</span><br/>
       {{data.credits}}c, {{data.ores}}o, {{data.knowledge}}k, {{data.qics}}q, [{{power('gaia')}}] {{power('area1')}}/{{power('area2')}}/{{power('area3')}} pw<br/>
-      Range: {{data.range}}, terraform level: {{data.terraformCostDiscount}}<br/>
+      Range: {{data.range}}, Terraform cost: {{3 - data.terraformCostDiscount}}<br/>
```

Comment:
It should be "Terraforming cost", and 'o' or 'ore' should be added at the end.

Reply:
I've tried to amend the commit, but it looks like it didn't work

Reply:
Try `git commit . --amend` (with the dot in the middle, at the root of the project)